docs/intro_tutorial_mli.md

```diff
@@ -27,6 +27,43 @@ There are four methods that need to be implemented:
     The old weighs of the model are discarded and replaced by the new weights.
 4. `current_weights` should return the current weights of the model.
 
+
+## Algorithms that work with colearn:
+These conditions need to be fulfilled for algorithms to work with collective learning:
+
+* The algorithm needs to have a "warm start", i.e. model fitting is incremental so that the previous model is 
```

Comment:
I would turn this around. i.e. Model fitting must be incremental, i.e. the algorithm needs to have a "warm start"